internal/Position.go

```diff
@@ -0,0 +1,7 @@
+package internal
+
+type Position struct {
+	ID     string  `json:"id"`
+	Name   string  `json:"name"`
+	Salary float64 `json:"salary"`
```

Comment:
You may lose accuracy if you use float type

Reply:
You tell about "ID" field?

Reply:
Type "number" is float32 in swagger?

Reply:
The type you using in open api spec is `number`. According to the openAPI standard it's ok to use it for precision values.

Reply:
I forgot about it(